src/OpenTelemetry.Api/Context/Propagation/ITextFormat.cs

```diff
@@ -49,5 +49,14 @@ public interface ITextFormat
         /// <param name="getter">Function that will return string value of a key with the specified name.</param>
         /// <returns>Span context from it's text representation.</returns>
         SpanContext Extract<T>(T carrier, Func<T, string, IEnumerable<string>> getter);
+
+        /// <summary>
+        /// Tests if an activity context has been injected into a carrier.
+        /// </summary>
+        /// <typeparam name="T">Type of object to extract context from. Typically HttpRequest or similar.</typeparam>
+        /// <param name="carrier">Object to extract context from. Instance of this object will be passed to the getter.</param>
+        /// <param name="getter">Function that will return string value of a key with the specified name.</param>
+        /// <returns><see langword="true" /> if the carrier has been injected with an activity context.</returns>
+        bool IsInjected<T>(T carrier, Func<T, string, IEnumerable<string>> getter);
```

Comment:
ITextFormatActivity had this, can I add to have the same effect?

Reply:
It's called by the Http instrumentation so we'll need it here if we want to remove `ITextFormatActivity`.

Reply:
I think, the correct question would be: is it ok to copy from ITextFormatActivity? Don't know exactly what the specs says

Reply:
It isn't part of the spec. I created an issue to try and fix that, not much progress made: https://github.com/open-telemetry/opentelemetry-specification/issues/674